package.json

```
@@ -6,6 +6,7 @@
   "scripts": {
     "build": "NODE_ENV=production webpack --config webpack.config.js --mode production",
     "format": "prettier --write .",
+    "lint": "eslint .",
```

```suggestion
    "lint": "eslint src/",
```

Because if you run `npm run build` or `npx tsc` both of these create temporary directories under the root (`html` and `tsbuild` respectively) which will then cause the linter to false fail. 

Either ensure all temporarily build directories are ignored; or (recommended) just lint the source code tree not other files.